sample-lnd.conf

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
nit: I think we can remove this line.

Reply:
No idea, where it come from. Was not intended and I will remove it.